SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,70 @@
+# Get started: 3 - Set up the Unreal GDK module and Starter Project
+
+Follow the steps below to:
+
+* Clone the Starter Project repository.
```

Comment:
These don't match the titles  - they need to match the titles to help me feel oriented.

Reply:
* Clone the Starter Project repository and the Unreak GDK repository

Reply:
i removed the list at the top
i dont think it's necessary to tell people what the steps will be as they can see the titles